### PR TITLE
language: fix: daml build used multiple thread by default

### PR DESCRIPTION
**Pull request by robin-da**
This fixes https://github.com/digital-asset/daml/issues/1180. `daml
build` was using multiple threads by default and as a consequence the
package id's became undeterministic.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs [hunk 1]
```diff
@@ -132,15 +132,19 @@ cmdBuild numProcessors =
     info (helper <*> cmd) $
     progDesc "Initialize, build and package the DAML project" <> fullDesc
   where
-    cmd = execBuild numProcessors <$> optionalOutputFileOpt
+    cmd =
+        execBuild <$> optionsParser numProcessors (pure Nothing) <*>
+        optionalOutputFileOpt
```
Comment (hurryabit) [inline daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs]: That is a slightly misleading way to break lines. Can we please do
```
func
  <$> arg1
  <*> arg2
```
Same below.

Comment (robin-da) [reply daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs]: it's autoformatted by hindent.

**Comment by neil-da** [comment]
Nice.

**Comment by neil-da** [comment]
Although why do we have `damlc build` and `damlc compile`? How do they differ? Any scope to unify them? `build` and `compile` are pretty much synonyms.

**Pull request merged**